cyprecice/cyprecice.pyx

```diff
@@ -1211,6 +1211,244 @@ cdef class Interface:
         self.thisptr.readScalarData (data_id, vertex_id, _value)
         return _value
 
+    def write_block_vector_gradient_data (self, data_id, vertex_ids, gradientValues):
+        """
+        Writes vector gradient data given as block. This function writes values of specified vertices to a dataID.
```

Comment:
```suggestion
        Writes vector gradient data given as block. This function gradient writes values of specified vertices to a dataID.
```

Reply:
Sounds a bit weird

Reply:
> Sounds a bit weird

Better now? A word was in the wrong place

Reply:
Awesome